openfisca_us/tests/policy/baseline/demographic/person/ccdf_eligibility.yaml

```diff
@@ -0,0 +1,54 @@
+- name: Asset eligible, age eligible, not enrolled, initial income eligible
+  period: 2022
+  input:
+    is_ccdf_asset_eligible: True
+    is_ccdf_age_eligible: True
+    is_enrolled_in_ccdf: False
+    is_ccdf_initial_income_eligible: True
+  output:
+    is_ccdf_eligible: True
+- name: Asset eligible, age eligible, not enrolled, initial income not eligible
```

Comment:
Could you add a line between tests? I think it could improve readability.

Reply:
I decided to have both values for each test since users would input the income, we can determine both initial and continuous eligibility from that. What do you think?